src/components/details-view/details-view.js

```diff
@@ -161,11 +169,13 @@ class DetailsView extends Component {
 
       // don't do these calculations when not scrolling the container
     } else if (e.currentTarget === e.target) {
+      const usageConsolidationHeight = this.$usageConsolidationContainer?.offsetHeight || 0;
+
       const top = e.currentTarget.scrollTop;
       const height = e.currentTarget.offsetHeight;
-      const scrollHeight = e.currentTarget.scrollHeight;
+      const scrollHeight = e.currentTarget.scrollHeight - usageConsolidationHeight;
```

Comment:
We need to subtract usage consolidation accordion height here, because we need to know where titles list end to know that we scrolled down to it